Secrets/KeysExample.swift

```diff
@@ -4,3 +4,8 @@ struct Keys {
         static let clientKey = ""
     }
 }
+
+func swiftLintTest() {
+    let someForceCast = NSNumber() as! Int
+    let colonOnWrongSide :Int = 0 
```

Comment:
Trailing Whitespace Violation (Medium Severity): Line #10 should have no trailing whitespace: current has 1 trailing whitespace characters<br>Colon Violation (Low Severity): When specifying a type, always associate the colon with the identifier